second-edition/src/ch08-01-vectors.md

```diff
@@ -129,33 +129,33 @@ let does_not_exist = &v[100];
 let does_not_exist = v.get(100);
 ```
 
-<span class="caption">Listing 8-6: Attempting to access the element at index
-100 in a vector containing 5 elements</span>
-
-When you run this code, the first `[]` method will cause a `panic!` because it
-references a nonexistent element. This method is best used when you want your
-program to consider an attempt to access an element past the end of the vector
-to be a fatal error that crashes the program.
-
-When the `get` method is passed an index that is outside the vector, it returns
-`None` without panicking. You would use this method if accessing an element
-beyond the range of the vector happens occasionally under normal circumstances.
-Your code will then have logic to handle having either `Some(&element)` or
-`None`, as discussed in Chapter 6. For example, the index could be coming from
-a person entering a number. If they accidentally enter a number that’s too
-large and the program gets a `None` value, you could tell the user how many
-items are in the current vector and give them another chance to enter a valid
-value. That would be more user-friendly than crashing the program due to a typo!
-
-#### Invalid References
-
-When the program has a valid reference, the borrow checker enforces the
-ownership and borrowing rules (covered in Chapter 4) to ensure this reference
-and any other references to the contents of the vector remain valid. Recall the
-rule that states we can’t have mutable and immutable references in the same
-scope. That rule applies in Listing 8-7 where we hold an immutable reference to
-the first element in a vector and try to add an element to the end, which won’t
-work:
+<span class="caption">Listando 8-6: Intentando acceder al elemento en el índice
+100 en un vector que contiene 5 elementos </ span>
+
+Cuando ejectures este código, el primer método `[]` causará `panic!` por que este
```

Comment:
_...Este método se usa mejor cuando quiere su programa para considerar un intento de acceder a un elemento más allá del final del vector sería un error fatal que bloquea el programa..._

Please consider revising the translated code, here the sentence can be better phrased (is hard to understand what is referring to)